tensorflow/java/src/test/java/org/tensorflow/GraphTest.java

```diff
@@ -71,6 +73,21 @@ private static void validateImportedGraph(Graph g, String prefix) {
   }
 
   @Test
+  public void getAllOperations() {
+    try (Graph g = new Graph()) {
+      List<Operation> operations = g.getOperations();
+
+      assertEquals(0, operations.size());
+
+      TestUtil.transpose_A_times_X(g, new int[2][2]);
```

Comment:
To keep the test tighter (i.e., not requiring the test reader to know what graph `transpose_A_times_X` will produce), how about changing this to add a single operation?
Something like:

```java
assertFalse(g.operations().hasNext());
Operation op = TestUtil.placeholder(g, "myPlaceholder", DataType.FLOAT).op();
Iterator<Operation> iter = g.operations();
assertTrue(iter.hasNext());
assertEquals(iter.next(), op);
assertFalse(iter.hasNext());
```